cs-multi-shell/src/app/route.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable, NgZone } from '@angular/core';
+import { NavController } from '@ionic/angular';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class RouteService {
+
+    constructor(private navController: NavController, private ngZone: NgZone) { }
+
+    public returnToLogin() {
+        this.ngZone.run(() => {
```

Comment:
Note: when events come back from plugins they are outside of the Angular zone - so change detection doesn't trigger. This is an example of how you may use ngZone if you were changing state of the application.